cypress/support/cypress-support/src/FireboltInvoker.js

```diff
@@ -91,6 +94,21 @@ export default class FireboltInvoker {
     logger.info('Firebolt Message in the invoke:' + JSON.stringify(fireboltMessage), 'invoke');
```

Comment:
comment in 97 can be changed to below
Saving firebolt event to an env map with the event name and listener id

Reply:
addressed the comment